examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -1183,25 +1183,36 @@ def main(args):
             text_encoder_one.gradient_checkpointing_enable()
             text_encoder_two.gradient_checkpointing_enable()
 
+    def get_lora_config(rank, use_dora, target_modules):
+        base_config = {
+            "r": rank,
+            "lora_alpha": rank,
+            "init_lora_weights": "gaussian",
+            "target_modules": target_modules,
+        }
+        
+        if use_dora:
+            base_config["use_dora"] = True
```

Comment:
This would fail for lower version of `peft` where `use_dora` isn't present in the call args.

Reply:
Thanks for pointing that out! Could you clarify which version of `peft` lacks `use_dora` in the call args?

Reply:
https://github.com/huggingface/diffusers/blob/13e8fdecda91e27e40b15fa8a8f456ade773e6eb/src/diffusers/loaders/peft.py#L220